prometheus.yml

```
@@ -0,0 +1,20 @@
+    metrics_path: '/actuator/prometheus' # Prometheus 가 메트릭을 수집할 경로
+    static_configs: # Prometheus 가 메트릭을 수집할 대상
+      - targets: # 컨테이너 이름 사용
+          - 'gateway-service:19090'
+          - 'userauth-service:19091'
+          - 'coinpay-service:19094'
+          - 'auction-service:19092'
+          - 'bidwin-service:19093'
+          - 'chat-service:19095'
+          - 'eureka-service:8761'
+
+############### 각 서버 설정하여 메트릭 수집해야할듯 #########################
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)